extension/llm/export/partitioner_lib.py

```diff
@@ -160,10 +160,8 @@ def get_qnn_partitioner(
             QnnPartitioner,
         )
 
-        # pyre-ignore: Undefined import [21]: Could not find a module corresponding to import `executorch.backends.qualcomm.serialization.qnn_compile_spec_schema`
-        from executorch.backends.qualcomm.serialization.qnn_compile_spec_schema import (
```

Comment:
Actually the error is 
```
ModuleNotFoundError: No module named 'executorch.backends.qualcomm.serialization.qnn_compile_spec_schema`
```
Seems like it's removed somewhere

Reply:
Many thanks for helping this!